cmake/onnxruntime_webassembly.cmake

```diff
@@ -225,6 +225,7 @@ else()
     "SHELL:-s EXPORT_ALL=0"
     "SHELL:-s VERBOSE=0"
     "SHELL:-s FILESYSTEM=0"
+    "SHELL:-s INCOMING_MODULE_JS_API=[preRun,locateFile,arguments,onExit,wasmMemory,buffer,instantiateWasm,mainScriptUrlOrBlob]"
```

Comment:
do we need to use the same list for ort-web to onnxruntime_test_all ?

Reply:
"mainScriptUrlOrBlob", for example, is not used by onnxruntime_test_all but it is required by ort-wasm. So perhaps need to figure out the full list of INCOMING_MODULE_JS_API per target.

Reply:
I can trim things down once ci is happy.